src/main/java/it/gov/pagopa/payhub/activities/dao/IuvSequenceNumberDao.java

```diff
@@ -0,0 +1,12 @@
+package it.gov.pagopa.payhub.activities.dao;
+
+public interface IuvSequenceNumberDao {
```

Comment:
Can you help me to locate this entity on the mapping document?
https://pagopa.atlassian.net/wiki/spaces/SPAC/pages/1406238722/Mapping+entit

Reply:
done

Reply:
Where have you reported this mapping? I cannot see it

Reply:
because i didn't publish the modifications... done now